swagger.yml

```diff
@@ -877,6 +944,62 @@ definitions:
         type: number
         description: The number of packages in the order.
         example: 3
+      packages:
```

Comment:
I don't think the tracking response gives back the package details. It just gives the package count